$OpenBSD: patch-source_build_include_compat_h,v 1.1 2018/07/10 13:53:29 naddy Exp $

Cast to int32_t to match the return type of the SSE2-optimized function
on amd64.  This fixes errors on other LP64-clang archs:
non-constant-expression cannot be narrowed from type 'long' to 'int32_t'

Index: source/build/include/compat.h
--- source/build/include/compat.h.orig
+++ source/build/include/compat.h
@@ -606,7 +606,7 @@ static FORCE_INLINE int32_t Blrintf(const float x)
     return n;
 }
 #else
-#define Blrintf lrintf
+#define Blrintf(x) static_cast<int32_t>(lrintf(x))
 #endif
 
 #if defined(__arm__)
